Fails to build due to an out-of-scope variable

I noticed when trying to make our "pulsar" docker build that this repository fails to build with the following error:
`helper.cpp:787:17: error: ‘all’ was not declared in this scope
  787 |                 all = PyList_New(0);
      |                 ^~~
`

--- a/helper.cpp
+++ b/helper.cpp
@@ -779,7 +779,8 @@ parallel.  See the individual functions for more details.");
 
 static int helper_exec(PyObject *module) {
 		import_array();
-		
+		PyObject *all;
+	
 		// Version information
 		PyModule_AddObject(module, "__version__", PyUnicode_FromString("0.1"));
 		

